cmd/cloudshell_open/appfile.go

```diff
@@ -32,9 +32,14 @@ type env struct {
 	Required    *bool  `json:"required"`
 }
 
+type scripts struct {
+	Postdeploy string `json:"postdeploy"`
```

Comment:
PostDeploy is a better variable name in this case.

Reply:
```suggestion
	PostDeploy string `json:"postdeploy"`
```